packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
 # Sentry Config File
```

Comment:
Weird behavior with passkeys: 

1 - When I click SignIn the passkeys selector it's prompted
2 - I select a passkey
3 - A different email will appear on the input

https://drive.google.com/file/d/1RSNgGsm15mNNWdSx8RHSwRlZpc2giZoj/view?usp=sharing

Reply:
@esaminu